packages/react-native/Libraries/Text/TextInput/Multiline/RCTUITextView.mm

```diff
@@ -132,6 +135,26 @@ - (void)textDidChange
   [self _invalidatePlaceholderVisibility];
 }
 
+- (void)setDisableKeyboardShortcuts:(BOOL)disableKeyboardShortcuts
+{
+  // Initialize the initial values only once
+  if (_isFirstRender) {
+    _isFirstRender = NO;
+    // Capture initial values of leading and trailing button groups
+    _initialValueLeadingBarButtonGroups = self.inputAssistantItem.leadingBarButtonGroups;
+    _initialValueTrailingBarButtonGroups = self.inputAssistantItem.trailingBarButtonGroups;
+  }
```

Comment:
This `isFirstRender` flag isn't the best idea.. 

You can set these flags just once in the initializer, right?

Reply:
Done - aa3fbcabf96853232d4ee359afe7942002e8154d, 374a7b50a06a7f6b89270167b40552539e114a5d.